packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -358,6 +366,31 @@ abstract class ImplicitlyAnimatedWidgetState<T extends ImplicitlyAnimatedWidget>
   /// [forEachTween] should override [didUpdateTweens] to update those
   /// properties. Dependent properties should not be updated within
   /// [forEachTween].
+  ///
+  /// {@tool sample}
```

Comment:
I think we have a policy about always placing a `#Sample Code` header above sample code, right?

Reply:
Also, it would be good to state under the header what this example is intended to show.

Reply:
I assumed `# Sample code` is just for classes and constructors but that's stupid. Will do.